website/cue/reference/components/sinks/blackhole.cue

```diff
@@ -32,7 +32,7 @@ components: sinks: blackhole: {
 	configuration: {
 		print_interval_secs: {
 			common:      false
-			description: "The number of seconds between reporting a summary of activity."
+			description: "The number of seconds between reporting a summary of activity. Set to 0 to disable reporting."
```

Comment:
Really minor nitpick, but shouldn't the value be enclosed in back-ticks?
```suggestion
			description: "The number of seconds between reporting a summary of activity. Set to `0` to disable reporting."
```

Reply:
Yeah that's a good call: https://github.com/vectordotdev/vector/pull/11915